### PR TITLE
Address review comments - remove `prepare.py` (load from csv files instead) - update README

### PR DESCRIPTION
**Pull request by ttngu207**

### CODE REVIEW

File: workflow_imaging/ingest.py [hunk 1]
```diff
@@ -1,65 +1,73 @@
 import scanreader
+import numpy as np
+import pandas as pd
```
Comment (dimitri-yatsenko) [inline workflow_imaging/ingest.py:3]: `pandas` is only used for reading a csv into a list of dicts, which can be accomplished with the `csv` module in the Python standard library. To reduce dependencies and increase stability, use the most standard solution: https://docs.python.org/3/library/csv.html#csv.DictReader

File: workflow_imaging/pipeline.py [hunk 2]
```diff
@@ -31,9 +31,15 @@
...
     definition = """
     -> subject.Subject
-    session_datetime: datetime
+    session_datetime: datetime(3)
     """
 
+    class Directory(dj.Part):
+        definition = """
+        -> master
+        session_dir: varchar(256)       # Relative path, relative to the `imaging_root_data_dir`
```
Comment (dimitri-yatsenko) [inline workflow_imaging/pipeline.py:40]: are there multiple session directories? In `get_scan_image_files` it is used as though there can be only one, retrieved with `fetch1`.

**Review by dimitri-yatsenko — CHANGES_REQUESTED** [review]
Some design questions.

File: workflow_imaging/pipeline.py (hunk 2 above)
Comment (ttngu207) [reply workflow_imaging/pipeline.py:40]: there is only one session directory per session, this is missing the `---`. Thanks for the catch Dimitri, fixing in next commits

File: workflow_imaging/ingest.py [hunk 3]
```diff
@@ -1,66 +1,77 @@
...
+    print(f'\n---- Insert {len(subjects_dict)} entry(s) into subject.Subject ----')
+    subject.Subject.insert(subjects_dict, skip_duplicates=True)
+
+
+def ingest_sessions():
+    root_data_dir = get_imaging_root_data_dir()
+
+    # ---------- Insert new "Session" and "Scan" ---------
+    with open('./user_data/sessions.csv', newline='') as f:
+        sessions_dict = list(csv.DictReader(f, delimiter=','))
```
Comment (dimitri-yatsenko) [inline workflow_imaging/ingest.py:25]: I would just call this `sessions`.  The variable name should describe the object it represents and not its datatype.

**Review by dimitri-yatsenko — APPROVED** [review]

File: workflow_imaging/ingest.py (hunk 3 above)
Comment (ttngu207) [reply workflow_imaging/ingest.py:25]: thanks @dimitri-yatsenko , changed to `input_sessions` to indicate that this is a user-input information (read from csv)

**Pull request merged**